frontend/src/layouts/AuthLayout.tsx

```diff
@@ -27,6 +27,7 @@ const AuthLayout: React.FC = ({ children }) => {
 
 const Content = styled.div`
   grid-area: content;
+  overflow: auto;
```

Comment:
the content region should never overflow, as this could push the footer out. Essentially the height dimensions of the header, footer, and navbar are fixed, and the content takes up the remaining space. If there is not enough space to display the content, scroll.